client/blocks/reader-post-card/style.scss

```diff
@@ -197,11 +197,11 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	width: 100%;
 
 	@media #{$reader-post-card-breakpoint-medium} {
-		margin-bottom: 10px;
+		padding-bottom: 10px;
```

Comment:
do any css gurus know if i could be introducing any bug by swapping a margin with a padding?